conda/recipes/versions.yaml

```diff
@@ -96,6 +96,8 @@ mypy_version:
   - '0.782'
 mysql_connector_cpp_version:
   - '8.0.23'
+nbsphinx_version:
+  - '>=0.8.6'
```

Comment:
Should we add a max version as well?

Reply:
If the patch version bump from `0.8.5` to `0.8.6` broke our builds, I'm not sure what would be a safe maximum version here :stuck_out_tongue:. Pinning to only `0.8.6` seems too restrictive. Probably fine to omit it for this package. We can add a maximum if it becomes a problem again.

Reply:
Sounds good :)